docs/platforms/react-native/migration/v5-to-v6.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: Migrate from 5.x to 6.x
+sidebar_order: 100
+description: "Learn about migrating from the SDK version 5 to 6"
+---
+
+The main goal of Sentry React Native SDK version 6 is bring compatibility with Sentry JavaScript version 8 and to improve our performance monitoring APIs and integrations. This version is breaking because of the JavaScript SDK dependency upgrade to version 8 and becuase we removed deprecated APIs, restructured npm package contents.
```

Comment:
```suggestion
The main goal of version 6 of the Sentry React Native SDK, is to bring compatibility with Sentry JavaScript version 8 and to improve our performance monitoring APIs and integrations. This version is breaking because of the JavaScript SDK dependency upgrade to version 8 and because we've removed deprecated APIs and restructured npm package contents.
```